### PR TITLE
Ember Data v1.13

### PR DESCRIPTION
**Pull request by locks**
**note** Work in Progress (WIP), do not merge

Started adding the snippets for the new Ember Data v1.13 stuff, like store methods and the new adapter.

## JavaScript
- [x] `store` methods
  - [x] find
  - [x] peek
  - [x] query
- [x] `JSONAPIAdapter`/`JSONAPISerializer`

## CoffeeScript
- [x] `store` methods
  - [x] find
  - [x] peek
  - [x] query
- [x] `JSONAPIAdapter`/`JSONAPISerializer`

## General
- [ ] review naming and snippets

### CODE REVIEW

File: snippets/functions.cson [hunk 1]
```diff
@@ -24,6 +16,41 @@
     """
 
   #
+  # Ember Data 1.0.0-beta
+  #
+
+  'Ember Data: Find':
```
Comment (locks) [inline snippets/functions.cson]: What do you think of the name @seifsallam? I prefixed all of the Ember Data things to make it more obvious.

Comment (seifsallam) [inline snippets/functions.cson]: :+1: That's better

**Comment by locks** [comment]
I **think** I've got everything, please let me know if I didn't.

**Comment by seifsallam** [comment]
@locks Awesome :+1:, can you just `queryRecord(type, {query})`

**Comment by locks** [comment]
Should be done :)

**Comment by seifsallam** [comment]
Thanks for the PR :)

**Pull request merged**